config.toml

```diff
@@ -0,0 +1,13 @@
+baseURL = "http://sistem.org/"
+languageCode = "en-us"
+title = "SISTEM"
+sectionPagesMenu = "main"
+
+[Params]
+  description = "a event ran by students, for students."
```

Comment:
`a` -> `An`

Reply:
i just stole this from the about section ;)